### PR TITLE
Cosmiture

### PR DESCRIPTION
**Pull request by Demadunk**
-Updated game guide and simplified nerd stats on weapons be more digestible for new readers. people who want to get in the fatty meat of it would go to the wiki
-Added a batch of double halloween candies for 2022.
-Added the gumball machine, you'll find it outside the 7/11 featuring real gacha mechanics.
-meet me in my next update where i code in tax forms for people to fill out their I-9's and W-4's in captcha-based tax writeoffs to get more slime
-Updated version number to befit current updates

### CODE REVIEW

**Review by C-rank — APPROVED** [review]
Looks good brochacho, runs well on test bot

File: ew/static/cfg.py [hunk 1]
```diff
@@ -3812,17 +3823,17 @@
...
-flavor_commands = "Command list: !salute\n!unsalute\n!hurl\n!howl\n!moan\n!pot\n!bully <target>\n!lol\n!jam <instrument>"
+miscellaneous_commands = "!scrutinize <object>: Examine specific objects in an area. Usually reserved for dungeons and ARGs.\n!shakeoff: If someone with the One Eye Open mutation is following you, use this to shake them off.\n!extractsoul: Remove your soul. from your body and bottle it.\n!returnsoul: Return your soul to your body, only if you have it in your inventory.\n!squeezesoul <soul>: Squeeze a soul. The soul's owner will vomit 1/4 of their slime on the ground.\n!ads: View ads in a district.\n!knock <player>: Knock on a player's apartment door, if you're in the district.\n!endlesswar: Check the total ammassed slime of all players.\n!negaslime: Check total amassed antislime."
+flavor_commands = "Command list: !harvest\n!salute\n!unsalute\n!hurl\n!howl\n!moan\n!pot\n!bully <target>\n!lol\n!jam <instrument>\n!measure <subject>\n!skate"
```
Comment (aristotlepenguin) [inline ew/static/cfg.py:3836]: stop being a fucking idiot.

File: ew/static/cosmetics.py [hunk 2]
```diff
@@ -1927,7 +1927,7 @@
     EwCosmeticItem(
         id_cosmetic = "yarmulke",
         str_name = "Yarmulke",
-        str_desc = "Oy Vey!", # For the love of all that is holy please do not smite me down, Igonis was the one behind this item.
+        str_desc = "Barukh Attah Adonai. May you keep your faith in your lord; the one true god, Yahweh", # Igonis is gone, I'm not keeping Oy Vey as the description text as ignois's request.
```
Comment (aristotlepenguin) [inline ew/static/cosmetics.py:1930]: why is igonis's flavor text verboten all of a sudden? or are you saying he asked it be changed? seems innocuous enough

File: ew/static/poi.py [hunk 3]
```diff
@@ -990,7 +990,7 @@
         role="NLAC U",
         permissions={'nlac-university': ['read', 'send', 'connect']},
         pvp=False,
-        vendors=['College'],
+        vendors=['College', "Gumball Machine"],
```
Comment (aristotlepenguin) [inline ew/static/poi.py:993]: we don't need to add a bunch of food vendors in new places. let's keep this in poudrin alley to complement the 7/11 instead

**Pull request merged**